purchasing/conductor/views.py

```diff
@@ -19,10 +22,11 @@
     ContractBase, ContractProperty, ContractStage, Stage,
     ContractStageActionItem
 )
+from purchasing.data.importer.costars import main
```

Comment:
I would alias this to make it a bit more readable. For example:

`from purchasing.data.importer.costars import main as import_costars`